src/Containers/MassTransit.AutoFacIntegration/RegistrationExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright 2007-2015 Chris Patterson, Dru Sellers, Travis Smith, et. al.
+//  
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the 
+// License at 
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0 
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR 
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the 
+// specific language governing permissions and limitations under the License.
+namespace MassTransit.AutofacIntegration

```

Comment:
yeah, I was going to fix that one :)



On Fri, Sep 18, 2015 at 12:14 PM, Travis Smith <notifications@github.com>
wrote:

> In src/Containers/MassTransit.AutoFacIntegration/RegistrationExtensions.cs
> <https://github.com/MassTransit/MassTransit/pull/345#discussion_r39890848>
> :
>
> > @@ -0,0 +1,38 @@
> > +﻿// Copyright 2007-2015 Chris Patterson, Dru Sellers, Travis Smith, et. al.
> > +//
> > +// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
> > +// this file except in compliance with the License. You may obtain a copy of the
> > +// License at
> > +//
> > +//     http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// Unless required by applicable law or agreed to in writing, software distributed
> > +// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
> > +// CONDITIONS OF ANY KIND, either express or implied. See the License for the
> > +// specific language governing permissions and limitations under the License.
> > +namespace MassTransit.AutofacIntegration
>
> This should be in the MassTransit namespace so it's available by default,
> no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MassTransit/MassTransit/pull/345/files#r39890848>.
>